tap_azure_tickets/__init__.py

```diff
@@ -217,30 +217,6 @@ def raise_for_error(resp, source, url):
     exc = ERROR_CODE_EXCEPTION_MAPPING.get(error_code, {}).get("raise_exception", AzureException)
     raise exc(message) from None
 
-def get_backoff_value_generator():
-    tries = 0
-    def backoff_value(response):
-        nonlocal tries
-        tries += 1
-        with suppress(TypeError, ValueError, AttributeError):
-            return int(response.headers.get("Retry-After"))
-        return 30 * (2 ** tries)
-
-    return backoff_value
-
-@backoff.on_exception(backoff.expo,
```

Comment:
These decorators are evaluated at compile time.  So, `get_backoff_value_generator` was created one time, which meant that all requests `tries` were the same variable.

So, if one request backed off 3 times and another backed off 2 times and another 4 times, then, the next request that needed to backoff would wait `30 * (2 ** 10) = 30720`.  It is pretty easy to see how a process could essentially hang by waiting forever.